commit 3dd87655fd92a375ada3fc6d8cfe90d431baecaa
Author: Andreas Boll <aboll@debian.org>
Date:   Mon Feb 11 14:55:34 2019 +0100

    Revert "Map evdev keycode KEY_FAVORITES to XF86Favorites"
    
    This reverts commit e6103610a2ddae633e2715c9f1da86e79b5256dd.
    
    This patch breaks keyboard layouts in XWayland applications when having
    several keyboard layouts enabled.
    
    See also https://bugzilla.redhat.com/show_bug.cgi?id=1587998
    
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921867
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921940
    Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=922020
    
    Drop this revert once we have a newer xkbcomp (>> 1.4.2) and bump
    dependency on x11-xkb-utils.

diff --git a/keycodes/evdev b/keycodes/evdev
index f2630ffe..51a6e26d 100644
--- a/keycodes/evdev
+++ b/keycodes/evdev
@@ -284,8 +284,6 @@ default xkb_keycodes "evdev" {
 	<I254> = 254;	// #define KEY_WWAN                246
 	<I255> = 255;	// #define KEY_RFKILL              247
 
-	<I372> = 372;   // #define KEY_FAVORITES           364
-
 	// Fake keycodes for virtual keys
 	<LVL3> =   92;
 	<MDSW> =   203;
diff --git a/symbols/inet b/symbols/inet
index 469a7d65..ef0635a7 100644
--- a/symbols/inet
+++ b/symbols/inet
@@ -218,8 +218,6 @@ xkb_symbols "evdev" {
     key <I254>   {      [ XF86WWAN              ]       }; // KEY_WWAN
     key <I255>   {      [ XF86RFKill            ]       }; // KEY_RFKILL
 
-    key <I372>  {       [ XF86Favorites          ]       };
-
     key <FK13>   {      [ XF86Tools         ]       };
     key <FK14>   {      [ XF86Launch5       ]       };
     key <FK15>   {      [ XF86Launch6       ]       };
